docs/qase.md

```diff
@@ -0,0 +1,21 @@
+## [Qase-Reporter](../pkg/qase/README.md)
+## [Qase-Patch-Validation-Create](#qase-patch-validation-create)
+
+
+# Qase-Patch-Validation-Create
```

Comment:
`Create Test Plans in Qase`

Reply:
This is not only for patch validation, is it?

Reply:
for now it is